### ISSUE TITLE
libzim binaries not installed for Windows

### ISSUE THREAD

**Issue by j0hnm4r5** [openzim/node-libzim#113]
I do see that you call out in the README, but obviously including the binaries for Windows would be ideal.

I opened up an issue over at [`@openzim/libzim`](https://github.com/openzim/libzim/issues/846), since it seems like building `libzim` for Windows is a pain point.

**Comment by j0hnm4r5** [comment]
Absolutely. I'll do everything I can.

**Comment by kelson42** [comment]
@j0hnm4r5 We have now the libzimbinaries for Windows, see https://download.openzim.org/release/libzim/libzim_win-x86_64-9.2.3-2.zip. We should build the binding!

**Comment by kelson42** [comment]
> @kelson42 do you remember what was missing in your #154 draft?

Not really. This is too old. Beside finishing the compilation script... the CI/CD should be done.

**Comment by Markus-Rost** [comment]
Looks like download-libzim.js hasn't been changed since your draft, so going fromm that draft should still be fine. However that also means anything that would be left to do is not something I'm very knowledgeable about.
